python/src/opendp/_convert.py

```diff
@@ -167,6 +170,9 @@ def _py_to_slice(value: Any, type_name: str) -> FfiSlicePtr:
     if type_name.startswith("Vec<") and type_name.endswith('>'):
         return _vector_to_slice(value, type_name)
 
+    if type_name.startswith("HashMap<") and type_name.endswith('>'):
```

Comment:
Did you cherry-pick these commits in? They're in the midst of your commit history. I usually have better luck when I rebase. 
`git rebase --onto [branch you want to sit on top of] [commit hash before where your commits start] [your last commit hash]`

I struggle with git management when my commits are not contiguous.

[See also](https://tworavens.slack.com/archives/C6M0ZSM1N/p1615501753004500)

Reply:
I didn't think I did, but I'm finding a lot of strange outcomes (repeated code in the notebook, for example) and I'm not sure what happened